packages/contract-wrappers/package.json

```diff
@@ -18,8 +18,8 @@
         "rebuild": "yarn wrappers:clean && yarn wrappers:generate && yarn wrappers:prettier && yarn build",
         "build:ci": "yarn build",
         "publish:private": "yarn rebuild && gitpkg publish",
-        "lint": "tslint --format stylish --project . --exclude **/lib/**/*",
-        "fix": "tslint --fix --format stylish --project . --exclude **/lib/**/*",
+        "lint": "eslint src --max-warnings 256",
```

Comment:
nit/optional: maybe lower this if the current number of warnings is lower?

Reply:
there are 0 warnings now that generated-* was excluded :)